pkg/time/mod.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Pulsate Epoch 2022 January 1st 00:00:0.000UTC
+ */
+export const EPOCH = 1640995200000n;
+declare const unixTimeFromEpochBase: unique symbol;
+export type UNIXTimeFromEpoch = number & {
+  [unixTimeFromEpochBase]: 'UNIXTimeFromEpoch';
+};
```

Comment:
```suggestion
declare const pulsateTime: unique symbol;
export type PulsateTime = number & {
  [pulsateTime]: never;
};
```
公称型にするためのフィールドの値に意味はないので, `never` にしておいてもよいかと思います. また型としては UNIX エポックではないので, UNIX の名前が含まれるのはよくないと思います.